csharp/TheRectangleClass/TheRectangleClass.Tests/Rectangle.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace TheRectangleClass.Tests
+{
+    [TestClass]
+    public class RectangleTests
+    {
+        [TestMethod]
+        public void Perimeter()
+        {
+            Rectangle rect = new Rectangle(3, 4);
+            Assert.AreEqual(14, rect.Perimeter());
+        }
+        [TestMethod]
+        public void Area()
+        {
+            Rectangle rect = new Rectangle(3, 4);
+            Assert.AreEqual(12, rect.Area());
+        }
+        [TestMethod]
+        public void IsSquareTrue()
```

Comment:
There are several naming convention for test names, the one I like the most is:
MethodName_Context_Expectation
for example this Test would be something like
_IsSquare_SameWidthAndHeight_ShouldReturnTrue_